packages/flutter/test/material/elevated_button_test.dart

```diff
@@ -375,6 +375,82 @@ void main() {
     expect(didLongPressButton, isTrue);
   });
 
+   testWidgets('ElevatedButton.icon onPressed and onLongPress callbacks are correctly called when non-null', (WidgetTester tester) async {
```

Comment:
Hey @Abhishek01039  these don't appear to test the change you are making. I don't think you will need to test callbacks for your change. Just adding a test that sets the related parameters to null and validates their values should be sufficient, similar to this test: https://github.com/flutter/flutter/blob/e422d5c724309eef3b8a24e8d9bc28943b04cc27/packages/flutter/test/material/elevated_button_test.dart#L14